core_impl/src/test/java/io/opencensus/stats/StatsTestUtil.java

```diff
@@ -53,108 +71,226 @@ private static StatsContextImpl createContext(
+      MutableAggregation mAggregation = MutableViewData.createMutableAggregation(aggregation);
+      for (double value : values) {
+        mAggregation.add(value);
+      }
+      mAggregations.add(mAggregation);
+    }
+
+    List<AggregationData> aggregates = new ArrayList<AggregationData>(aggregations.size());
+    for (MutableAggregation mAggregation : mAggregations) {
+      aggregates.add(MutableViewData.createAggregationData(mAggregation));
     }
-    MutableDistribution.Range range = mdist.getRange();
-    return DistributionAggregate.create(
-        mdist.getCount(),
-        mdist.getMean(),
-        mdist.getSum(),
-        DistributionAggregate.Range.create(range.getMin(), range.getMax()),
-        tags,
-        mdist.getBucketCounts());
+    return aggregates;
   }
 
   /**
-   * Creates a {@code DistributionAggregate} by adding the given values to a new {@link
-   * MutableDistribution} that does not have {@code BucketBoundaries}.
+   * Compare the {@code MutableAggregation} with expected values based on its underlying type. The
+   * expected values are all optional (nullable).
    *
-   * @param tags the {@code DistributionAggregate}'s tags.
-   * @param values the values to add to the distribution.
-   * @return the new {@code DistributionAggregate}
+   * @param aggregation the {@code MutableAggregation} to be verified.
+   * @param sum the expected sum value, if aggregation is a {@code MutableSum}.
+   * @param count the expected count value, if aggregation is a {@code MutableCount}.
+   * @param bucketCounts the expected bucket counts, if aggregation is a
+   *        {@code MutableHistogram}.
+   * @param min the expected min value, if aggregation is a {@code MutableRange}.
+   * @param max the expected max value, if aggregation is a {@code MutableRange}.
+   * @param mean the expected mean value, if aggregation is a {@code MutableMean}.
+   * @param stdDev the expected standard deviation, if aggregation is a
+   *        {@code MutableStdDev}.
+   * @param tolerance the tolerance used for {@code double} comparison.
+   * @throws AssertionError if {@code MutableAggregation} doesn't match with expected value.
    */
-  static DistributionAggregate createDistributionAggregate(
-      List<Tag> tags, List<Double> values) {
-    MutableDistribution mdist = MutableDistribution.create();
-    for (double value : values) {
-      mdist.add(value);
-    }
-    MutableDistribution.Range range = mdist.getRange();
-    return DistributionAggregate.create(
-        mdist.getCount(),
-        mdist.getMean(),
-        mdist.getSum(),
-        DistributionAggregate.Range.create(range.getMin(), range.getMax()),
-        tags);
+  static void assertMutableAggregationEquals(
+      MutableAggregation aggregation, final Double sum, final Long count, final long[] bucketCounts,
+      final Double min, final Double max, final Double mean, final Double stdDev,
+      final double tolerance) {
```

Comment:
I just realized this method is only used at `MutableViewDataTest.testCreateMutableAggregation()`. Since that test is updated, I just removed this method.